src/plugins/visualize/public/application/components/visualize_editor_common.tsx

```diff
@@ -75,6 +78,41 @@ export const VisualizeEditorCommon = ({
   embeddableId,
   visEditorRef,
 }: VisualizeEditorCommonProps) => {
+  const opensearchDashboards = useOpenSearchDashboards<VisualizeServices>();
+  const { keyboardShortcut } = opensearchDashboards.services;
+
+  const handleSidebarToggle = useCallback(() => {
+    const sidebarButton = document.querySelector(VISUALIZE_DOM_SELECTORS.SIDEBAR_TOGGLE_BUTTON);
+    if (sidebarButton) {
+      (sidebarButton as HTMLElement).click();
+    }
+  }, []);
+
+  const handleSave = useCallback(() => {
+    const saveButton = document.querySelector(VISUALIZE_DOM_SELECTORS.SAVE_BUTTON);
+    if (saveButton) {
+      (saveButton as HTMLElement).click();
+    }
+  }, []);
+
+  keyboardShortcut?.useKeyboardShortcut({
+    id: 'toggle_vis_sidebar',
+    pluginId: 'visualize',
+    name: 'Toggle Visualization Sidebar',
+    category: 'Panel / layout',
+    keys: 'shift+c',
+    execute: handleSidebarToggle,
+  });
+
+  keyboardShortcut?.useKeyboardShortcut({
+    id: 'save_visualization',
+    name: 'Save Visualization',
+    pluginId: 'visualize',
+    category: 'Editing / Save',
```

Comment:
It looks like almost (or all?) shortcuts have category that is different from each other. Wouldn't this make the help modal look very busy?